zuul.d/base.yaml

```diff
@@ -35,6 +35,7 @@
       - .gitignore
       - .golangci.yaml
       - .pre-commit-config.yaml
+      - test
```

Comment:
either `tests?/` as a proper regexp, or at least group "tests" and "test"

Note that we have `- tests` on Line 27, but also `^tests/.*$` on line 20 - sounds like duplications.